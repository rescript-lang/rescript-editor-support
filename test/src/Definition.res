let xx = 10

let y = xx
//      ^def

module Inner = {
  type tInner = int
  let vInner = 34
}

type typeInner = Inner.tInner
//                     ^def