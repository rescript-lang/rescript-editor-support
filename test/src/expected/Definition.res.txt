Definition src/Definition.res 2:8
{"uri": "Definition.res", "range": "{"start": "{"line": "0", "character": "4"}", "end": "{"line": "0", "character": "6"}"}"}

